src/main/java/com/github/tomakehurst/wiremock/common/xml/XmlNode.java

```diff
@@ -97,17 +97,17 @@ public XmlNode(Node domNode) {
     attributes =
         domNode.hasAttributes()
             ? convertAttributeMap(domNode.getAttributes())
-            : Collections.<String, String>emptyMap();
+            : Collections.emptyMap();
   }
 
   private static Map<String, String> convertAttributeMap(NamedNodeMap namedNodeMap) {
-    ImmutableMap.Builder<String, String> builder = ImmutableMap.builder();
+    Map<String, String> map = new HashMap<>();
     for (int i = 0; i < namedNodeMap.getLength(); i++) {
       Node node = namedNodeMap.item(i);
-      builder.put(node.getNodeName(), node.getNodeValue());
+      map.put(node.getNodeName(), node.getNodeValue());
     }
 
-    return builder.build();
+    return map;
```

Comment:
Can we preserve immutability here e.g. by calling Collections.unmodifiableMap()?